command/server/listener.go

```diff
@@ -97,6 +97,15 @@ func listenerWrapTLS(
 		}
 		tlsConf.PreferServerCipherSuites = preferServer
 	}
+	if v, ok := config["tls_require_valid_client_cert"]; ok {
```

Comment:
That's a tricky question. I don't think there's value in most of the client auth options -- like, why allow setting to check client auth if given but ignore it if it doesn't pass? Because this is the only option that seems reasonable, I figured I'd just turn it into a bool (in the config it's a set of possible consts).